examples/ds_config.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Do we need  "msamp": { "enabled": false }? Maybe it would be more clear.

Reply:
Good idea. Added.